doc/userguide/rules/smb-keywords.rst

```diff
@@ -58,3 +58,48 @@ Examples::
 ``smb.ntlmssp_domain`` is a 'sticky buffer'.
 
 ``smb.ntlmssp_domain`` can be used as ``fast_pattern``.
+
+
+smb.version
+------------
+
+Keyword to match on SMB version.
```

Comment:
I think it should mention somewhere that the match is on the suricata smb tx, not on every smb record on the wire.

Reply:
Good point, will update in next PR